enc/encode_parallel.cc

```diff
@@ -216,6 +219,28 @@ bool WriteMetaBlockParallel(const BrotliParams& params,
 
 }  // namespace
 
+std::pair<size_t, size_t>  consecutiveMapping(const size_t tid, const size_t nThreads, const size_t nElements){
+
+    const size_t elementsPerThread = ceil((float) nElements / nThreads);
```

Comment:
should be `ceil(static_cast<double>(nElements) / nThreads);` to avoid float overflows